src/components/Coaching/CoachingList.tsx

```diff
@@ -0,0 +1,77 @@
+import { Box, Divider, styled, Typography } from '@material-ui/core';
+import { EcoOutlined } from '@material-ui/icons';
+import { Skeleton } from '@material-ui/lab';
+import React, { ReactElement } from 'react';
```

Comment:
Suuuuper unimportant comment here, just my preference/opinion but I always like to have ```react``` be the top import for the file. Just keeps things consistent. It's fine as is 🙂